src/blocks/subscribe/index.php

```diff
@@ -54,6 +54,15 @@ function enqueue_scripts() {
 }
 add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\\enqueue_scripts' );
 
+/**
+ * Generate a unique ID for each registration form.
+ *
+ * @return string A unique ID string to identify the form.
+ */
+function get_form_id() {
+	return \wp_unique_id( 'newspack-subscribe-' );
```

Comment:
The main intent is that the person analyzing analytics should be able to tell what _type_ of form was submitted—registration vs. subscription—whether or not it was submitted in a prompt, and which prompt. So it shouldn't really matter that the ID isn't consistent across page views as long as you can infer what type of form it is. I used form ID mainly because AMP Analytics provides an easy placeholder to populate that value on submit events.

Reply:
That makes it clearer :) Can we avoid calling it an id, though? I image this might confuse any future developers, including our future selves.

Reply:
2e38a7f adds an explanation of what we're using the ID for, and why it doesn't need to be consistent across page views. In the future, we could improve this function to make it more consistent and predictable, but that seems like an unnecessary complication to me at the moment.